platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java

```diff
@@ -138,8 +137,9 @@ private static void validateMapbox() {
    * @param accessToken the access token to validate
    * @return true is valid, false otherwise
    */
-  private static boolean isAccessTokenValid(String accessToken) {
-    return !(TextUtils.isEmpty(accessToken)
-      || (!accessToken.toLowerCase(MapboxConstants.MAPBOX_LOCALE).startsWith("pk.")));
+  static boolean isAccessTokenValid(String accessToken) {
+    return !(accessToken == null || accessToken.length() == 0)
```

Comment:
We could also trim the string.

Reply:
done in https://github.com/mapbox/mapbox-gl-native/pull/12458/commits/45b60690f314c2ca6eb93174ad0ef9a3911cee98